res/default-favicon.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <text style="font: italic 13px sans-serif" x="8" y="12.5" text-anchor="middle">h</text>
```

Comment:
In the future, I'd like to replace the "h" (for "http") by an incremented number, so that we'd have a different icon for different hostnames, even when they don't have a favicon.